lib/electrode-server.js

```diff
@@ -223,6 +223,7 @@ module.exports = function electrodeServer(appConfig, decors, callback) {
     const configOptions = {
       dirs: [Path.join(__dirname, "config")],
       warnMissing: false,
+      failMissing: false,
```

Comment:
turn off error if electrode-server doesn't have a config file for the value of NODE_ENV